templates/request_notification.html

```diff
@@ -0,0 +1,12 @@
+<p>Hi {{ reciever_name }},</p>
+<br>
+<p>Greetings from Mentorship System!</p>
+<br>
+<p>You have recieved a mentorship relation request: </p>
+<br>
+<p><b>{{ sender_name }} ({{ sender_email }})</b> wants to be your <b>{{ sender_role }}</b> until <b>{{ end_date }}</b></p>
```

Comment:
```suggestion
<p><b>{{ sender_name }} ({{ sender_email }})</b> wants to be your <b>{{ sender_role }}</b> until <b>{{ end_date }}</b></p>
```

I'm not sure if the email of the user should be displayed for privacy purposes 🤔 this can be changed later... in fact this made me think about this new feature [I wrote on Slack](https://systers-opensource.slack.com/archives/CAE8QK41L/p1576275723003100)

You don't have to remove for now, but it's worth mentioning.